packages/cli/docs-preview/src/previewDocs.ts

```diff
@@ -104,7 +104,7 @@ class ReferencedAPICollector {
 
         const dbApiDefinition = convertAPIDefinitionToDb(
             apiDefinition,
-            id,
+            ir.apiDefinitionId ?? id,
```

Comment:
hmm I think we'll want to keep this `id` otherwise its possible local preview may break

Reply:
Will remove, I wasn't positive if we'd need it but can def nix